node/chainSimulator/configs/configs.go

```diff
@@ -268,9 +271,8 @@ func generateValidatorsKeyAndUpdateFiles(
 	nodes.RoundDuration = args.RoundDurationInMillis
 	nodes.StartTime = args.GenesisTimeStamp
 
-	// TODO fix this to can be configurable
-	nodes.ConsensusGroupSize = 1
-	nodes.MetaChainConsensusGroupSize = 1
```

Comment:
This is a very cumbersome approach to "fix" this or refactor this TODO on this feat branch for several reasons:
- You refactor/fix a todo from master/rc branch inside a feat branch, which will most likely be merged in months, if not years into master => you **need to always maintain new test** that are coming from main branch to this feat.
- It will literally kill us after each merge rc -> feat
- If someone also starts fixing this, it will cause a big merge conflict => lots of overhead for nothing

This "fix" should only happen in a fix branch targeted to rc/v1.7.next1 then merge that into feat/chain-go-sdk